src/deltacode/__init__.py

```diff
@@ -75,12 +98,12 @@ def align_scans(self):
         which calls utils.align_trees().
         """
         try:
-            utils.fix_trees(self.new.files, self.old.files)
+            self.new_files_original_path , self.old_files_original_path = utils.fix_trees(self.codebase1, self.codebase2)
```

Comment:
I am not sure we need this anymore now that we are using VirtualCodebase

Reply:
In fact, I think we can throw all the stuff having to do with path alignment and original paths out. 

We should treat the VirtualCodebases as the are loaded and not try to manipulate paths.

Reply:
@majurg we must align the path as we are making delta comparison based upon it.
as I explained here https://github.com/nexB/deltacode/pull/167/files#r611008180

Virtual Codebase does not take care of the stipped of path after alignment.
Now if we are not having the aligned path whenever we change the root directory(which we supporrt currently) even the files which are moved will be difficult to trace. So we are keeping that align scans

Reply:
Now if we are not planning to have the allign scans I think we should also treat  files like `old/a1.py`   and `new/a1.py`   as added even if they are having the same sha1.

But we don't do it currently and they are treated as same.

Reply:
> We should treat the VirtualCodebases as the are loaded and not try to manipulate paths.

We are doing the path manipulation to compare the deltas.